src/app/add-history/page.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import Header from '@/components/layout/Header';
+import { useEffect } from 'react';
+import { usePostStore } from '@/store/usePostStore';
+import AICard from './_components/AICard';
+import MenuList from './_components/MenuList';
+import DeliveryDate from './_components/DeliveryDate';
+import CButton from '@/components/Button';
+import Title from './_components/Title';
+import CostInput from './_components/CostInput';
+
+export default function AddHistoryPage() {
+    const { setUpdateMenu } = usePostStore();
+
+    useEffect(() => {
+        setUpdateMenu(null);
+    }, []);
+
+    return (
+        <div className="p-4">
```

Comment:
```suggestion
        <div className="p-4 pb-40">
```
<img width="403" height="876" alt="스크린샷 2025-07-14 오전 9 55 58" src="https://github.com/user-attachments/assets/35e75898-7419-4d44-802b-e7244bfd8ae6" />

- 모바일 크기에서 하단 플로팅 버튼이 겹쳐지고 있어서 하단에 padding 을 주거나 다른 방법을 사용해서 공백을 주는 건 어떨까요?

Reply:
좋습니다!!

Reply:
안되는 부분 작성했던 기능은 아직 수정은 안했습니다